components/scripts/lib/java.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
+invoke_java() {
```

Comment:
0: This isn't something to do right now, but I had originally borrowed this code from the launch scripts that the Gradle Application Plugin generates.  But it has a number of problems.  The BT Team recently updated the application plugin to generate a much better launch script. At some point, we should update this code to be based on that.

Again, this is good for now though.

Reply:
I suggest to create a GH issue for this update.

Reply:
Issue created: https://github.com/gradle/gradle-enterprise-build-validation-scripts/issues/176